portal/views/notification.py

```diff
@@ -0,0 +1,159 @@
+"""Views for UserNotifications"""
+from flask import abort, jsonify, Blueprint, current_app
+from ..database import db
+from ..extensions import oauth
+from ..models.notification import UserNotification
+from ..models.user import current_user, get_user
+from .portal import check_int
+
+
+notification_api = Blueprint('notification_api', __name__, url_prefix='/api')
+
+
+@notification_api.route('/user/<int:user_id>/notification')
+@oauth.require_oauth()
+def get_user_notification(user_id):
+    """Retrieve Notifications for the given User
+    ---
+    tags:
+      - Notification
+      - User
+    operationId: getUserNotification
+    parameters:
+      - name: user_id
+        in: path
+        description: TrueNTH user ID
+        required: true
+        type: integer
+        format: int64
+    produces:
+      - application/json
+    responses:
+      200:
+        description: successful operation
+        schema:
+          id: user_notifications
+          properties:
+            notifications:
```

Comment:
I think we typically wrap multiple resources in a Bundle, are we still doing that? @pbugni